Integrations/GitHub/GitHub.py

```diff
@@ -0,0 +1,277 @@
+                                                  'body': r['body'],
+                                                  'state': r['state']}}
+        else:
+            print_results['HumanReadable'] = 'No such issue'
+        print_results['Contents'] = r
+        demisto.results(print_results)
+    else:
+        demisto.results({
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['text'],
+            'Contents': r,
+            'HumanReadable': 'Please enter a valid issue number',
+        })
+    return r
+
+
+def close_issue(issue_num, state):
+    r = None
+    if str.isdigit(str(issue_num)):
+        print_results = {
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['markdown']
+        }
+        r = http_request('GET', 'search/issues?q=repo:teizenman/Demisto-start+' + str(issue_num))
+        if r['total_count'] > 0:
+            query = {'state': state}
+            search = json.dumps(query)
+            r = http_request('POST', 'repos/teizenman/Demisto-start/issues/' + str(issue_num), search)
+            print_results['HumanReadable'] = 'Issue #' + str(issue_num) + ' has been closed'
+            print_results['EntryContext'] = {
+                'GitHub.Issues(val.ID==obj.ID)': {'ID': str(r['id']), 'state': 'close'}
+            }
+        else:
+            print_results['HumanReadable'] = 'No such issue'
+        print_results['Contents'] = r
+        demisto.results(print_results)
+    else:
+        demisto.results({
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['text'],
+            'Contents': r,
+            'HumanReadable': 'Please enter a valid issue number',
+        })
+    return r
+
+
+def make_search(search_query, sort, order):
+    query = '?q=repo:teizenman/Demisto-start'
+    if search_query:
+        query += '+' + search_query
+    if sort:
+        query += '&sort=' + sort
+        if order:
+            query += '&order=' + order
+    r = http_request('GET', 'search/issues' + query)
+    print_results = {
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['markdown'],
+        'Contents': r
+    }
+    if r['total_count'] != 0:
+        results = []
+        for issue in r['items']:
+            issue_details = makehash()
+            issue_details['ID'] = str(issue['id'])
+            issue_details['title'] = issue['title']
+            issue_details['link'] = issue['url']
+            issue_details['state'] = issue['state']
+            results.append(issue_details)
+            print_results['HumanReadable'] = tableToMarkdown(
+                str(r['total_count']) + ' found issues.\nComplete list of issues: ' + str(not r['incomplete_results']),
+                results, ['ID', 'title', 'state', 'link'])
+        print_results['EntryContext'] = {
+            'GitHub.Searches(val.query==obj.query)': {'query': query, 'results': results}
```

Comment:
Non plural please, this is the convention
```suggestion
            'GitHub.Search(val.query==obj.query)': {'query': query, 'results': results}
```